src/ui.loader-rendering/index.vue

```diff
@@ -1,7 +1,12 @@
 <template>
   <Transition v-bind="config.transition">
     <div v-if="showLoader" v-bind="wrapperAttrs">
-      <ULoader :loading="showLoader" size="lg" :color="color === 'white' ? 'grayscale' : 'white'" />
+      <ULoader
+        v-bind="loaderAttrs"
```

Comment:
**v-bind** should be the last one to be able to change props passed by `config` prop.